android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -648,6 +648,20 @@ class DisplayToolbar internal constructor(
     internal fun removeNavigationAction(action: Toolbar.Action) {
         views.navigationActions.removeAction(action)
     }
+
+    /**
+     * Hides menu button in display mode.
```

Comment:
```suggestion
     * Hides the menu button in display mode.
```

There's a couple more places where this is applicable.